packages/cli-kit/src/node/ruby.ts

```diff
@@ -9,7 +9,7 @@ import {AdminSession} from '../session.js'
 import {content, token} from '../output.js'
 import {Writable} from 'node:stream'
 
-const RubyCLIVersion = '2.20.1'
+const RubyCLIVersion = '2.23.0'
```

Comment:
I'm assuming this will be the version containing [this CLI2 PR](https://github.com/Shopify/shopify-cli/pull/2558), which is needed for this one.